orchestration/src/test/java/com/sap/ai/sdk/orchestration/OrchestrationUnitTest.java

```diff
@@ -281,14 +234,31 @@ void filteringStrict() {
         post(urlPathEqualTo("/v2/inference/deployments/abcdef0123456789/completion"))
             .willReturn(jsonResponse(response, SC_BAD_REQUEST)));
 
-    final var config = FILTERING_CONFIG.apply(NUMBER_0);
+    final var filter = createAzureContentFilter(NUMBER_0);
 
-    assertThatThrownBy(() -> client.chatCompletion(config))
+    assertThatThrownBy(() -> client.chatCompletion(prompt, config.withFilteringConfig(filter)))
```

Comment:
```suggestion
    final var configWithFilter = config.withFilteringConfig(filter);

    assertThatThrownBy(() -> client.chatCompletion(prompt, configWithFilter))
```
It's now clear which method throws